infrastructure/kube/keep-test/monitoring/prometheus-service.yaml

```
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus
+  namespace: monitoring
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '9090'
+spec:
+  selector:
+    app: prometheus
+  type: NodePort
+  ports:
+    - port: 9090
+      targetPort: 9090
+      nodePort: 30000
```

Can we use `prometheus` as we named the port in the deployment?

```suggestion
      nodePort: prometheus
```